npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -161,10 +161,14 @@ def inferred_npmrc # rubocop:disable Metrics/PerceivedComplexity
         @inferred_npmrc = nil
       end
 
+      def npm_version
+        Helpers.npm_version_numeric(package_lock.content)
```

Comment:
newbie q: I'm assuming we don't guess the `npm` version like we do for `pnpm` and `yarn`?

Reply:
We do some guessing based on the lockfile format, but we still don't respect the `packageManager` setting in `package.json`. We probably should!